src/main/java/com/desiato/puresynth/models/UserDTO.java

```diff
@@ -0,0 +1,37 @@
+package com.desiato.puresynth.models;
+
+public class UserDTO {
```

Comment:
record

Reply:
ok, thanks for the clarification. Now I am creating a new UserDTO and passing the data